applications/callflow/src/module/cf_group_pickup_feature.erl

```diff
@@ -107,22 +108,26 @@ build_pickup_params(_ ,'undefined', _) ->
 build_pickup_params(_, Other, _) ->
     {'error', <<Other/binary," not implemented">>}.
 
--spec params_from_data(kz_term:ne_binary(), kz_json:object(), kapps_call:call()) ->
+-spec params_from_data(kz_term:ne_binary(), kz_json:object(), kz_json:object()) ->
                               {'ok', kz_term:proplist()} |
                               {'error', kz_term:ne_binary()}.
-params_from_data(<<"user">>, Data, _Call) ->
+params_from_data(<<"user">>, Data, _Flow) ->
     EndpointId = kz_json:get_ne_binary_value(<<"id">>, Data),
     {'ok', [{<<"user_id">>, EndpointId}]};
-params_from_data(<<"device">>, Data, _Call) ->
+params_from_data(<<"device">>, Data, _Flow) ->
     EndpointId = kz_json:get_ne_binary_value(<<"id">>, Data),
     {'ok', [{<<"device_id">>, EndpointId}]};
-params_from_data(<<"ring_group">>, Data, _Call) ->
+params_from_data(<<"ring_group">>, Data, _Flow) ->
     [Endpoint |_Endpoints] = kz_json:get_list_value(<<"endpoints">>, Data, []),
     EndpointType = kz_json:get_ne_binary_value(<<"endpoint_type">>, Endpoint),
     {'ok', [{<<EndpointType/binary, "_id">>, kz_doc:id(Endpoint)}]};
-params_from_data(<<"page_group">>, Data, _Call) ->
-    params_from_data(<<"ring_group">>, Data, _Call);
+params_from_data(<<"page_group">>, Data, _Flow) ->
+    params_from_data(<<"ring_group">>, Data, _Flow);
 params_from_data('undefined', _, _) ->
     {'error',<<"module not defined in callflow">>};
-params_from_data(Other, _, _) ->
-    {'error',<<"module ", Other/binary, " not implemented">>}.
+params_from_data(_Other, _, Flow) ->
```

Comment:
Personally I would like a clause that handles when `Flow` is `undefined`. I know the code is coping as-is but would prefer explicit handling to avoid future readers' confusion.